gen/kneeboard.py

```diff
@@ -248,7 +256,8 @@ def __init__(
         self.comms.append(CommInfo("Flight", self.flight.intra_flight_channel))
 
     def write(self, path: Path) -> None:
-        writer = KneeboardPageWriter()
+        is_night_mission = self.start_time.hour > 19 or self.start_time.hour < 7
```

Comment:
This needs to be an option. Good for the shift+k kneeboard, but players that read the kneeboard that's actually on their leg probably want it to be white for night missions.

Reply:
Good point, I forgot you could read kneeboard that way. I'll set it up as an option (maybe with a small caption indicating that it could make the kneeboard hard to read if used on the pilot body ?)

Reply:
Sounds good. I suppose another option would be to generate both, but having twice as many kneeboard pages would make it really annoying to sift through.

Aside from that the patch looks ready to merge.

Reply:
I've updated the PR, now comes with a toggle in game settings, with disclaimer when using kneeboard on pilot body. I've also modified the base color for the "light theme" kneeboard to have a slightly less aggressive contrast.